torch/nested/_internal/ops.py

```diff
@@ -1074,3 +1115,71 @@ def _nested_get_ragged_idx(func, *args, **kwargs):
 
     inp = new_kwargs.pop("input")
     return inp._ragged_idx
+
+
+@register_jagged_func(
+    torch.ops.aten._nested_strided_to_jagged.default, "self: t, dummy: jt_all"
+)
+def _nested_strided_to_jagged(func, *args, **kwargs):
+    _, new_kwargs = normalize_function(
+        func, args=args, kwargs=kwargs, normalize_to_only_use_kwargs=True
+    )
+
+    inp: torch.Tensor = new_kwargs.pop("input")
+
+    # All jagged NT can be converted into strided NTs, but the opposite is not True
+    # Only strided NTs with a single jagged dimension might be converted into
+    # jagged NTs, so first we check for that
+    ragged_dims_count = 0
+    ragged_idx = -1
+    for i in range(inp.dim()):
+        try:
+            inp.size(i)
+        except RuntimeError:
+            ragged_dims_count += 1
+            ragged_idx = i
+    assert (
+        ragged_dims_count == 1
+    ), "Only strided NTs with 1 jagged dim can be converted to jagged NT"
```

Comment:
extreme nit: I think we generally refer to "ragged" dims in error messages
```suggestion
    ), "Only strided NTs with 1 ragged dim can be converted to jagged NT"
```